javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -526,30 +547,22 @@ private Vector(int indexShift, HashArrayMappedTrie<Integer, T> trie) {
     public Vector<T> drop(long n) {
         if (n <= 0) {
             return this;
-        }
-        if (n >= length()) {
+        } else if (n >= length()) {
```

Comment:
I've not understood yet why we call a method `length()` when the instance variable `length` is in scope. Isn't it additional overhead to perform internal abstractions?